src/Infrastructure.EntityFramework/AdminConsole/Configurations/OrganizationEntityTypeConfiguration.cs

```diff
@@ -12,9 +12,18 @@ public void Configure(EntityTypeBuilder<Organization> builder)
             .Property(o => o.Id)
             .ValueGeneratedNever();
 
+        // Deprecated https://bitwarden.atlassian.net/browse/PM-10863
         builder.Property(c => c.LimitCollectionCreationDeletion)
             .ValueGeneratedNever()
-            .HasDefaultValue(true);
+            .HasDefaultValue(false);
```

Comment:
I thought EF generated its properties based on the `Entity` class - what does this configuration class do? Is it required to set the default value?

Reply:
Yeah, it seems like this is mostly just to set the default value. I don't think it's necessary if the default is `false` - so I've removed these on the latest changes.

Reply:
Now the column is non-nullable with no default value though - that's going to throw errors when you remove it from the code but haven't removed it from the db yet. (Even in EF we persist the column for an additional release as a shadow property after it's removed from the Entity class.) Bools will default to `false` in C# but I think you still need the default value for the database itself.

Reply:
That isn't something this code was doing. [The migrations that included these configuration references](https://github.com/bitwarden/server/commit/5e231f770d7c87a9b543463b8ba20614160def63#diff-66eb2e465dc54617072b9061c488289da865a0e77a0dbd22d384ea0c719351ceR13-R35) still resulted in `nullable: false` database columns. I'm not sure we really want nullable database columns here - that is never really a valid state for the data in EF land.

To keep the column around for a release event after we remove the field from code we would want a shadow property, but that should still go on the entity. On that change I'll just reverse the way [these values](https://github.com/bitwarden/server/pull/4709/files#diff-dc77aef59f6aee693dac228d795b9af1254ad275970d861bffbc4c5888d52effR96-R97) mirror each other so that the old field gets/sets from the new one. I'd also move the declaration of the properties to the EF specific entity model to avoid still trying to pass it to Dapper. The release after will remove the old property from the entity (removing it from EF) and remove it from SQL. 

Does that sound right?

Reply:
I think it's a good move to use shadow properties for this change - it confirms that the first stage of the EDD cycle contains db changes only.

I don't think I wanted it to be nullable, more that I thought it needed to have a default value. But I see how you can control that through the shadow property definition, which is neat. So you've addressed my concerns here.